master/internal/api_checkpoint.go

```diff
@@ -28,6 +37,59 @@ func (a *apiServer) GetCheckpoint(
 	}
 }
 
+func (a *apiServer) DeleteCheckpoints(
+	ctx context.Context,
+	req *apiv1.DeleteCheckpointsRequest) (*apiv1.DeleteCheckpointsResponse, error) {
+	deleteCheckpoints := req.CheckpointUuids
```

Comment:
I don't think this tmp variable makes this clearer to read (maybe `checkpointUUIDsToDelete`, but then it is too long)